src/_cffi_src/openssl/cryptography.py

```diff
@@ -25,6 +25,8 @@
 #include <windows.h>
 #endif
 
+#define CRYPTOGRAPHY_OPENSSL_101_OR_GREATER \
+    (OPENSSL_VERSION_NUMBER >= 0x1000100 && !CRYPTOGRAPHY_IS_LIBRESSL)
```

Comment:
We only support OpenSSL 1.0.1+ so this should always be true for OpenSSL. Additionally, it looks like `SSL_export_keying_material` is available in LibreSSL so I don't think you actually need to use a conditional binding at all here.

Reply:
Makes sense. I assume that means I should remove the changes to `src/cryptography/hazmat/bindings/openssl/_conditional.py` as well?

Reply:
Yeah you should only need the function definition and none of the other boilerplate.